### DISCUSSION TITLE
Add a typegen template for making typegen issue reproductions

### DISCUSSION THREAD

**Discussion by johtso** [statelyai/xstate#3268] [Ideas]
Would it be possible to make a codesandbox (or similar) template that generates typegen types?

Would be real handy for opening issues as currently I'm fiddling around doing the error prone thing of pasting the typegen interface manually into the viz.

**Comment by mattpocock** [comment]
Should be possible with Stackblitz

**Reply by johtso** [reply]
Was thinking maybe Stackblitz!
